app/messages/Messages.scala

```diff
@@ -352,6 +352,11 @@ object Messages {
       welsh   = "Dysgu sut i gofrestru"
     )
 
+    val `If you need to speak to an adviser call us on 0300 200 3835 at the Business Support Service to talk about your payment options.`: Message = Message(
```

Comment:
Where's this new content come from? I can't see it in the prototype v3.1

Reply:
Second paragraph:
https://supp-prototype.herokuapp.com/3-1/unhappy/not-enrolled
https://supp-prototype.herokuapp.com/3-1/vat/not-enrolled

Reply:
I feel like it should be one message or the other, and maybe prototype isn't synced up - both messages are basically the same just different word ordering within the sentence. Also, can we get a ticket for this? I don't want to start merging in content changes without a ticket as it's technically not properly testable - we can't just say "check protoytype"

We need a ticket saying, it was "this" it should be "this" and list each page

Reply:
Also in Welsh spreadsheet: https://docs.google.com/spreadsheets/d/1CdBoO5ASk4iwsvHStPkT36tbIZNduLOC/edit#gid=47327513

Reply:
I will create a ticket & double-check exactly which wording with Rocco. But I am following the spreadsheet of content in ticket 579.